src/utils.rs

```diff
@@ -707,25 +707,26 @@ impl<'a, 'b> fmt::Display for Emoji<'a, 'b> {
 }
 
 fn str_width(s: &str) -> usize {
-    #[cfg(feature = "unicode-width")]
+    #[cfg(feature = "unicode-display-width")]
     {
-        use unicode_width::UnicodeWidthStr;
-        s.width()
+        unicode_display_width::width(s) as usize
```

Comment:
Note: this usize cast is here to avoid changing APIs. It may not be what is wanted?

Reply:
This seems okay to me. Would sure be nice to avoid breaking compatibility for this.